blog/tutorial/01-index.md

```diff
@@ -395,11 +407,11 @@ var (
 
 ```
 
-## Launch
+## Launch the Application
```

Comment:
```suggestion
## Launch the App
```

Reply:
let's use app consistently 
see the word list https://developers.google.com/style/word-list#letter-a 
> app
> In general, use app instead of application when referring to programs for end users, especially in the context of mobile or web software. In some contexts, such as enterprise software, it's OK to use application to convey a sense of greater complexity. Use application in standard phrases such as application programming interface.
>